addons/native_dialog/gtk_filesel.c

```diff
@@ -46,6 +46,10 @@ static gboolean create_gtk_file_dialog(gpointer data)
 
    _al_gtk_make_transient(display, window);
 
```

Comment:
> [...] amongst others.

Is there any publicly available documentation on that?  I'd like to avoid future stylistic mismatches.